default.py

```diff
@@ -204,32 +209,41 @@ def clean(self, video_type):
 
         return cleaned_files, count, self.exit_status
 
-    def clean_all(self):
+    def clean_all(self, **kwargs):
         """
         Clean up any watched videos in the Kodi library, satisfying any conditions set via the addon settings.
 
         :rtype: (str, int)
         :return: A single-line (localized) summary of the cleaning results to be used for a notification, plus a status.
         """
         debug("Starting cleaning routine.")
-
+        video_type=kwargs.get('video_type', None)
+        expired_videos=kwargs.get('expired_videos', None)
+        
+        
         if get_setting(clean_when_idle) and xbmc.Player().isPlaying():
             debug("Kodi is currently playing a file. Skipping cleaning.", xbmc.LOGWARNING)
             return None, self.exit_status
 
         results = {}
         cleaning_results, cleaned_files = [], []
-        if not get_setting(clean_when_low_disk_space) or (get_setting(clean_when_low_disk_space) and disk_space_low()):
+        if (expired_videos != none) or (not get_setting(clean_when_low_disk_space) or (get_setting(clean_when_low_disk_space) and disk_space_low())):
```

Comment:
This is one of the lines that triggered a warning in my IDE. You did not capitalize ``None`` and as such you are comparing it to the (nonexistent) variable ``none``, which will most likely result in a runtime error. Also, comparing with ``None`` is not necessary. Any value other than the empty dict ``{}`` will evaluate to ``True`` so you can just write ``if expired_videos:``